src/css/components/_captions-settings.scss

```diff
@@ -21,6 +21,48 @@
   vertical-align: bottom;
 }
 
+// code that will only run if CSS Grid is supported by the browser
+@supports (display: grid) {
+  .vjs-text-track-settings .vjs-modal-dialog-content {
+    display: grid;
+    grid-template-columns: 1fr auto;
+    grid-template-rows: 1fr auto;
+  }
+
+  .vjs-text-track-settings .vjs-track-settings-colors {
+    display: block;
+    grid-column: 1;
+    grid-row: 1;
+  }
+
+  .vjs-text-track-settings .vjs-track-settings-font {
+    grid-column: 2;
+    grid-row: 1;
+  }
+
+  .vjs-text-track-settings .vjs-track-settings-controls {
+    grid-column: 2;
+    grid-row: 2;
+    text-align: unset;
+    vertical-align: unset;
+  }
+  .vjs-text-track-settings fieldset span.vjs-opacity {
+    margin-left: unset;
+  }
+  .video-js {
+    @media (max-width: 365px) {
+      .vjs-modal-dialog .vjs-modal-dialog-content {
```

Comment:
we shouldn't set generic modal-dialog properties inside this file. We should target the text-track-settings specifically.

Reply:
Also, this doesn't seem to be getting applied?

Reply:
I had this in there to resize the font if the player was a certain size. but your are correct doesn't seem to work, nor make a beneficial difference in appearance when I just tested manually